src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String welcome;
+    private int variable;
```

Comment:
What does `variable` mean?

Reply:
The name of the variable isn’t correct, because in this variable we save the current year. Maybe I should call it as `currentYear`?